integration-tests/src/test/java/org/apache/druid/tests/indexer/AbstractIndexerTest.java

```diff
@@ -69,6 +73,16 @@ protected Closeable unloader(final String dataSource)
 
   protected void unloadAndKillData(final String dataSource)
   {
+    // Get all failed task logs
+    List<TaskResponseObject> allTasks = indexer.getCompleteTasksForDataSource(dataSource);
+    for (TaskResponseObject task : allTasks) {
+      if (task.getStatus().isFailure()) {
+        LOG.info("------- Found failed task. Start log:");
+        LOG.info(indexer.getTaskLog(task.getId()));
```

Comment:
How about showing a certain amount of logs at the end instead of printing the whole? It will likely kill the Travis job by hitting the max number of logs per job. You can add a `offset` query parameter when you call the Overlord API.

Reply:
Any suggestion on the offset value? Another idea I have is to get the errorMsg field in the report API call

Reply:
Maybe the offset could be `max(88000, 80% of total log size)`, where 88000 is an approximate size of the startup logs.

Reply:
Or we can do offset=-16000 which is the same as the Console UI Log window popup thing

Reply:
Oh yeah, forgot that it can be a negative. I think it should be a bit larger than that though. I usually don't find anything interesting from the logs which the UI shows by default.

Reply:
Set the offset to -88000 and also added a log of the errorMsg so that we are sure we will definitely have the error message